src/com/facebook/buck/apple/AppleBundle.java

```diff
@@ -670,6 +672,24 @@ public boolean isLegacyWatchApp() {
     return stepsBuilder.build();
   }
 
+  private void verifyResourceConflicts(AppleBundleResources resources,
+                                       SourcePathResolver resolver) {
+    // Ensure there are no resources that will overwrite each other
+    Set<Path> filenames = new HashSet<>();
+    for (SourcePath path : Iterables.concat(
+        resources.getResourceDirs(),
```

Comment:
It seems that this is duplicating the resource paths form lines 427-429:

    resources.getResourceDirs(),
    resources.getDirsContainingResourceDirs(),
    resources.getResourceFiles()

We should extract those and pass them into the method instead.